src/SimpleDetailsList.tsx

```diff
@@ -1,21 +1,23 @@
 import * as React from 'react';
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import {
+  ContextualMenu,
+  DirectionalHint,
+  IContextualMenuItem,
+  IContextualMenuProps
+} from 'office-ui-fabric-react/lib/ContextualMenu';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';

```

Comment:
notice here i'm importing directly from the roots; if you import everything from the module, then your bundle will get everything exported from the module. Even with webpack 2 treeshaking. Only rollup.js will do treeshaking properly at the moment.